### ISSUE TITLE
Error on install

### ISSUE THREAD

**Issue by RyanRoberts** [billythekid/prodder#1]
Just downloaded and tried to install the plugin, but it instantly borked with the following error:

```
PHP warning

array_filter() expects parameter 1 to be array, null given

/Users/*********/Documents/Beanstalk/dtgroup/craft/plugins/prodder/services/ProdderService.php(80)
```

![screen shot 2016-09-13 at 10 59 52](https://cloud.githubusercontent.com/assets/108874/18469727/1f2f31b4-79a1-11e6-9f9e-1770c69d2aef.png)

**Comment by billythekid** [comment]
Confirmed reproducible, install on vanilla craft with devMode turned on.

**Comment by billythekid** [comment]
Fixed in 1.0.3
